config/ant-phase-verify.xml

```diff
@@ -118,6 +118,7 @@
 
           <!-- Exclude because printed AST is verified by matching checkstyle output -->
           <exclude name="**/grammar/**/Input*.txt"/>
+          <exclude name="**/grammar/**/Expected*.txt"/>
```

Comment:
It took me a while to figure this is why violations were showing up but I wasn't finding a matching suppression.

Reply:
All original inputs should be renamed now so this item is all expected.